test-one/.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Should this folder be committed to source control?

Reply:
I'm not to sure about that. I never used that file and didn't know it was there until showing the hidden folders in Visual Studio.